src/store/reducers/overview/overview.ts

```diff
@@ -51,7 +51,7 @@ export const overviewApi = api.injectEndpoints({
                 }
             },
             keepUnusedDataFor: 0,
-            providesTags: ['All'],
+            providesTags: ['All', 'SchemaTree'],
```

Comment:
Overview and Schema are still affected by Diagnostics manual refresh, since the same apis are used in Diagnostics. I did not find a simple solution that prevents it, so I would leave it as before, since it does not look as a big issue